### ISSUE TITLE
plot_contour not working with multi-objective study

### ISSUE THREAD

**Issue by Akkarine** [optuna/optuna#2613]
`_get_contour_plot()` at `optuna/visualization/_contour.py` is not compatible with multi-objective study.

## Environment

- Optuna version: 2.7.0
- Python version: 3.8.5
- OS: ubuntu 20.04.2

## Error messages, stack traces, or logs

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-48-3ccdd3f1173a> in <module>
----> 1 optuna.visualization.plot_contour(study=study, target=value_to_display, target_name=target_name, params=['daily_seasonality','weekly_seasonality'])

~/.local/share/virtualenvs/jupyterlab-NhNFbejC/lib/python3.8/site-packages/optuna/visualization/_contour.py in plot_contour(study, params, target, target_name)
     88     _imports.check()
     89     _check_plot_args(study, target, target_name)
---> 90     return _get_contour_plot(study, params, target, target_name)
     91 
     92 

~/.local/share/virtualenvs/jupyterlab-NhNFbejC/lib/python3.8/site-packages/optuna/visualization/_contour.py in _get_contour_plot(study, params, target, target_name)
    159         y_param = sorted_params[1]
    160         sub_plots = _generate_contour_subplot(
--> 161             trials, x_param, y_param, study.direction, param_values_range, target, target_name
    162         )
    163         figure = go.Figure(data=sub_plots, layout=layout)

~/.local/share/virtualenvs/jupyterlab-NhNFbejC/lib/python3.8/site-packages/optuna/study.py in direction(self)
    125 
    126         if self._is_multi_objective():
--> 127             raise RuntimeError(
    128                 "A single direction cannot be retrieved from a multi-objective study. Consider "
    129                 "using Study.directions to retrieve a list containing all directions."

RuntimeError: A single direction cannot be retrieved from a multi-objective study. Consider using Study.directions to retrieve a list containing all directions.
```

**Comment by nzw0301** [comment]
Hi @Akkarine, I think the bug has been fixed by #2589, but this patch will be in v2.8.0 that is not released yet. :( 
Could you use the master branch?

**Comment by Akkarine** [comment]
It's exactly my case. I'll wait for release :)
